diff --git i/PKGBUILD w/PKGBUILD
index cc199eb..436ac0c 100644
--- i/PKGBUILD
+++ w/PKGBUILD
@@ -15,7 +15,7 @@ pkgdesc="Debugging, in-system programming and boundary-scan testing for embedded
 arch=('i686' 'x86_64' 'arm' 'aarch64')
 url="http://openocd.org"
 license=('GPL')
-depends=('libftdi-compat' 'libusb-compat' 'hidapi' 'libudev.so' 'capstone' 'libjaylink')
+depends=('libftdi-compat' 'libusb-compat' 'hidapi' 'capstone' 'libjaylink' 'jimtcl')
 makedepends=('git' 'automake>=1.11' 'autoconf' 'libtool' 'tcl')
 options=(!strip)
 provides=('openocd')
@@ -23,19 +23,10 @@ conflicts=('openocd')
 
 source=(
   "${pkgname}::git+https://repo.or.cz/openocd.git"
-  "git+https://github.com/msteveb/jimtcl.git"
-  "git+https://gitlab.zapb.de/libjaylink/libjaylink.git"
-  "git+https://git.savannah.nongnu.org/git/git2cl.git"
-
 )
-md5sums=('SKIP'
-         'SKIP'
-         'SKIP'
-         'SKIP')
-sha1sums=('SKIP'
-          'SKIP'
-          'SKIP'
-          'SKIP')
+sha256sums=(
+  'SKIP'
+)
 
 # Specify desired features and device support here. A list can be
 # obtained by running ./configure in the source directory.
@@ -86,11 +77,6 @@ pkgver() {
 prepare() {
   cd "$srcdir/${pkgname}"
   sed -i 's|GROUP="plugdev", ||g' contrib/60-openocd.rules
-  git submodule init
-  git config submodule.jimtcl.url "$srcdir/jimtcl"
-  git config submodule."src/jtag/drivers/libjaylink".url "$srcdir/libjaylink"
-  git config submodule."tools/git2cl".url "$srcdir/git2cl"
-  git -c protocol.file.allow=always submodule update
 }
 
 build() {
